pootle/apps/pootle_app/management/commands/update_tmserver.py

```diff
@@ -94,8 +94,25 @@ def __init__(self, *args, **kwargs):
     def get_units(self, *filenames):
         """Gets the units to import and its total count."""
         units = []
+        all_filenames = set()
 
         for filename in filenames:
+            if not os.path.exists(filename):
+                continue
```

Comment:
You can use `self.stdout.write()` and friends if you init the parser objects with a reference to the stdout handler from the command.